core/src/main/java/com/graphhopper/routing/weighting/custom/SpeedCalculator.java

```diff
@@ -55,12 +55,16 @@ public SpeedCalculator(final double maxSpeed, CustomModel customModel, DecimalEn
             Object value = entry.getValue();
 
             if (value instanceof Number) {
+                double number = ((Number) value).doubleValue();
+                if (number > maxSpeed)
```

Comment:
This needs to be done for the speed_factor too ... will add it later as it is required to refactor this a bit more to introduce the "range bucketing".